daemon/inertiad/project/deployment_test.go

```diff
@@ -118,7 +118,8 @@ func TestGetStatusIntegration(t *testing.T) {
 		builder:   fakeBuilder,
 	}
 	status, err := deployment.GetStatus(cli)
-	assert.NoError(t, err)
+
+	assert.Nil(t, err)
```

Comment:
the right way to check for no error is:

```suggestion
	assert.NoError(t, err)
```

see #545